src/Templates/src/templates/maui-blazor/Platforms/MacCatalyst/Info.plist

```diff
@@ -2,6 +2,15 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+    <!-- The App store requires you specify if the app uses encryption. -->
```

Comment:
```suggestion
    <!-- The Mac App Store requires you specify if the app uses encryption. -->
```

(At a minimum the `S` needs to be capitalized, but I suggest also prefixing `Mac`.)